### PR TITLE
K8sConfiguration .Net SDK - Create

### PR DESCRIPTION
**Pull request by NarayanThiru**
Create .net SDK for K8sConfiguration

### CODE REVIEW

File: sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.KubernetesConfiguration</PackageId>
+    <Description>Provides Microsoft Azure KubernetesConfiguration management operations including the ability to create, update and delete runbooks and schedules.</Description>
+    <Version>3.8.2-preview</Version>
```
Comment (isra-fel) [inline sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj:9]: Suggest `1.0.0-preview.1` for a new module. See https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md

File: sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+  </PropertyGroup>
+
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime" Version="2.3.21" />
+    <PackageReference Update="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Update="System.Net.Http" Version="4.3.4" />
+  </ItemGroup>
```
Comment (isra-fel) [inline sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj:31]: Please remove these references, as they are already defined in `AzSdk.reference.props`

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Comment by isra-fel** [comment]
@NarayanThiru Hello, please also add a link to the openapi spec review PR here
Thanks

File: sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj (hunk 1 above)
Comment (NarayanThiru) [reply sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj:9]: Fixed.

File: sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj (hunk 2 above)
Comment (NarayanThiru) [reply sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj:31]: Removed

**Comment by NarayanThiru** [comment]
API Spec is at https://github.com/Azure/azure-rest-api-specs/tree/master/specification/kubernetesconfiguration

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**